back/node_watcher/datamodel.prisma

```diff
@@ -180,3 +180,20 @@ type BlockIndex {
   startFrom: Int!
   index: Int!
 }
+
+type Treasury {
```

Comment:
this should probably be named `TreasurySpendProposal` to avoid confusion

Reply:
but what else is in treasury other than spen proposals

Reply:
well the treasury would have an avaialable balance, and spend period, for example

Reply:
also this struct has a treasuryProposalId, which the Treasury in general wouldn't have

Reply:
done